File: JavaIOStudy/app/src/main/java/com/example/javaiostudy/MainActivity.java
package com.example.javaiostudy;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;

import java.io.File;
import java.io.IOException;

public class MainActivity extends AppCompatActivity {

    private static final String TAG = "MainActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        try {
            File tempFile = File.createTempFile("temp", ".tmp");
            Log.d(TAG, "onCreate: " + tempFile.getAbsolutePath());
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: FluidColorfulFrame/app/src/main/java/com/example/fluidcolorfulframe/MainActivity.kt
package com.example.fluidcolorfulframe

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.fluidcolorfulframe.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }
}